CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
-
+- v0.23.0
+  - Drop support for PyPy 3.7 and 3.8. ([#470](https://github.com/PyO3/rust-numpy/pull/470))
```

Comment:
We probably should have added an entry for `Element: Sync`, maybe just fixup here (or as a follow-up)?

Reply:
Agreed 👍, will include it in #457, haven't written the entry for that one yet.